mlocal/frags/Makefile.stub

```diff
@@ -20,7 +20,9 @@ collect:
 check: $(go_OBJ)
 	@echo " CHECK golangci-lint"
 	$(V)cd $(SOURCEDIR) && \
-		golangci-lint run --build-tags "$(GO_TAGS)" ./...
+		( [ -x bin/golangci-lint ] || \
```

Comment:
would it be possible to change this to something like `command -v golangci-lint > /dev/null || [ -x bin/golangci-lint ] || curl ...` so that if the user already has a golangci-lint installation, the build won't try to fetch another copy? This is nice for things like distributions, that manage dependencies in a different way.

Or push the check (and installation?) of golangci-lint to mconfig?

Reply:
Ok @mem I changed it as you suggested.